fedimint-server/src/config/mod.rs

```diff
@@ -307,11 +305,13 @@ impl ServerConfig {
                 local,
                 private,
                 consensus,
+                consensus_json,
             } = config;
 
             self.local.modules.insert(name, local);
             self.private.modules.insert(name, private);
             self.consensus.modules.insert(name, consensus);
+            self.consensus.modules_json.insert(name, consensus_json);
```

Comment:
Kind of weird to have this duplicated, would it be possible to custom implement `serde` to produce the json versions?

Reply:
Not in two separate keys, I think. And it would be an extra complexity. It might be a bit weird, but is there only for human convenience.